src/main/java/org/folio/service/orders/utils/StatusUtils.java

```diff
@@ -28,16 +28,22 @@ public static boolean areAllPoLinesCanceled(List<PoLine> poLines) {
     return poLines.stream().allMatch(StatusUtils::isStatusCanceledPoLine);
   }
 
-  public static boolean changeOrderStatus(PurchaseOrder purchaseOrder, List<PoLine> poLines) {
+  public static boolean changeOrderStatusForOrderUpdate(PurchaseOrder purchaseOrder, List<PoLine> poLines) {
```

Comment:
we should not transit from Closed to Open workflowStatus in case when using API even if PO Line has some uncomplete status, related karate test: https://github.com/folio-org/folio-integration-tests/blob/e0ef0143cd7c72ab2b61fcbb933615169a7470b7/acquisitions/src/main/resources/thunderjet/mod-orders/features/close-order-including-lines.feature#L4

Reply:
UI for closing the order never includes composite PO Lines, so deleting toBeReopened() has not influence in UI, but API after this change become more transperent